devel/protobuf3-cpp-upstream/Portfile

```diff
@@ -60,7 +60,11 @@ worksrcdir      protobuf-${release_version}
 # So don't let the project cmake add zlib; already added (last) by base.
 patchfiles-append cmake-zlib-include.diff
 
-compiler.cxx_standard   2011
+# Revert a breakage from dropping mach_time in:
+# https://github.com/protocolbuffers/protobuf/commit/a8a9bd2e4990fd5e0f0098bdfbe62bc00783eab7
+patchfiles-append patch-unbreak-time.diff
```

Comment:
I suspect this works fine on modern systems? If so, please make the patch conditionally.

Reply:
There is no need in that. Firstly, that was a recent change done for the sake of iOS. It was not due to earlier code not working. Secondly, this is a development port with no dependents. It is supposed to be experimental. Thirdly, I have commented there in the merged PR in upstream regarding the breakage, if it goes unnoticed, I open a new issue for that. Once upstream hopefully resolved it, I will update the patch.